src/System.Net.Security/tests/FunctionalTests/NegotiateStreamStreamToStreamTest.cs

```diff
@@ -16,8 +16,13 @@ namespace System.Net.Security.Tests
     [PlatformSpecific(TestPlatforms.Windows)] // NegotiateStream only supports client-side functionality on Unix
     public abstract class NegotiateStreamStreamToStreamTest
     {
+        private const int PartialBytesToRead = 5;
         private readonly byte[] _sampleMsg = Encoding.UTF8.GetBytes("Sample Test Message");
 
+        private const int MaxWriteDataSize = 63 * 1024; // NegoState.MaxWriteDataSize
+        private static string s_longString = new string('A', MaxWriteDataSize) + 'Z';
+        private readonly byte[] _longMsg = Encoding.ASCII.GetBytes(s_longString);
```

Comment:
Will remove those unused properties (they will be useful in next PR to improve more code coverage though)